plone/app/upgrade/v30/tests.py

```diff
@@ -661,10 +662,20 @@ def testAddEmailCharsetProperty(self):
             self.assertEqual(self.portal.getProperty('email_charset'), 'utf-8')
 
     def testUpdateMemberSecurity(self):
+        # These properties were removed in Plone 5, so we add them
+        # manually here and check if they are properly updated by the
+        # updateMemberSecurity upgrade step
         pprop = getToolByName(self.portal, 'portal_properties')
+        self.portal.manage_addProperty('validate_email', False, 'boolean')
```

Comment:
@jcerjak Don't we have to check if that property already exist? I would expect this to fail if we would run it in Plone 4.x.